packages/codemods/src/bin/spirit-codemods.js

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env node
+// eslint-disable-next-line import/no-unresolved -- The import is relative to the `dist` root
+import { cli } from '../index.js';
```

Comment:
```suggestion
import { cli } from '..';
```

?

Reply:
This is run directly by the node, so no parsers like Babel or Typescript which will take care of default imports.

Reply:
yes, this have to stay as it is